### DISCUSSION TITLE
Way to associate pick object with player object?

### DISCUSSION THREAD

**Discussion by nathan-j-hart** [cwendt94/espn-api#222] [Q&A] [answered]
I'm wondering if there is a way to find the player object associated with picks from a past draft? I would like to get more information on each pick e.g. position. Any ideas?

**Comment by nathan-j-hart** [comment, marked as answer]
Found the answer, feeding the player name and ID into the player_info() function of the league class.

**Reply by cwendt94** [reply]
Nice, that is correct!
